contracts/ics/base/src/handler.rs

```diff
@@ -61,6 +61,7 @@ pub fn navigate_packet() -> Result<Navigator> {
         MsgType::MsgConsumeAckPacket => {
             Ok(Navigator::CheckMessage(envelope, AxonClient::default()))
         }
+        MsgType::MsgRecvPacket => Ok(Navigator::Skip),
```

Comment:
Why should this be skipped?

Reply:
because MsgRecvPacket is handled by channel contract, and packet has no duty to do it, but when we use useless WriteAck packet as input cell to save capacity, packet contract will be triggered and throw error